doc/manual/manual.tex

```diff
@@ -7795,6 +7795,43 @@ \subsubsection{The van Keken thermochemical composition benchmark}
 
 We can observe that when the two-layer viscosity model is added, there is only one apparent peak for each case. The first peaks of the 0.01 viscosity contrast tests appear earlier and are larger in magnitude than those of 0.1 viscosity contrast tests.  There are no secondary plumes and the whole system tends to reach stability after around 500 time steps.
 
+
+\subsubsection{The Bunge et al. mantle convection experiments}
+
+\textit{This section was contributed by Cedric Thieulot and Bob Myhill.}
+
+Early mantle modelling studies of the 70's, 80's and 90's were often concerned with simple setups (Cartesian geometries, incompressible fluid, free slip boundary conditions) and investigated the influence of the Rayleigh number, the heating mode or the temperature dependence of the viscosity on temperature, pressure and/or strain rate \cite{youn74,buss75,buss79,BBC89,BC93,vavy93,burb97}. In this cookbook, we use the `simple' material model to reproduce the setup in \cite{burb96}, which reported that even modest increases in mantle viscosity with depth could have a marked effect on the style of mantle convection.
+
+Although the original article showcases results obtained in a 3D hollow sphere, we here run the models in an annular domain of inner radius R$_i$=3480 \si{\kilo\meter} and outer radius  R$_o$ = 6370 \si{\kilo\meter}.
+The surface temperature is set to $T_{\textit{surf}}$ = 1060 \si{\kelvin} and 
+the bottom temperature to $T_{\textit{cmb}}$ = 3450 \si{\kelvin}
+The gravity vector is radial and its magnitude is $g$ = 10 \si{\metre\per\second\squared}.
+
+There is a single incompressible fluid in the domain, characterised by $\rho_0$ = 4500 \si{\kilogram\per\metre\cubed}, $\alpha$ = $2.5\cdot10^{-5}$ \si{\per\kelvin}, 
+$k$ = 4 \si{\watt\per\metre\per\kelvin},
+$C_p$ = 1000 \si{\joule\per\kilogram\per\kelvin} and
+its internal heating rate is $Q_{\textit{int}}$ = $1\cdot10^{-12}$ \si{\watt\per\kilogram}.
```

Comment:
Here and elsewhere:
```suggestion
its internal heating rate is $Q_{\text{int}}$ = $1\cdot10^{-12}$ \si{\watt\per\kilogram}.
```
The point is specifically that you want that text to *not* show up as italics, but rather as upright text because it's an abbreviation of natural language *text*, not formula variables (which should be shown as italics).

Reply:
Huh, I always thought of the subscript as a part of the variable (because otherwise Q_int and Q_ext would be the same formula variable). But mostly I just dislike the kerning when the subscript is upright.

Reply:
New discovery: \textsubscript and \textsuperscript can be used both inside and outside of the math environment (and they both use upright typeface).

Reply:
The typographic community says that if you refer to matrix entries, use `A_{ij}`, which prints both `i` and `j` in italics because they are variable names, and puts a very small amount of space between the two of them to indicate that they are separate symbols. On the other hand, if you have a matrix `A_{stress}`, it should be written as `A_\text{stress}` to indicate that you don't have separate indices `s`, `t`, `r`, `e`, `s`, `s` that either serve as separate indices or a product of them, but rather that you are using a natural-language word. `\text` marks things up in this way: as *text*, rather than *variables in a formula*.